jobs/v4/howto/basic_job_sample.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	talent "cloud.google.com/go/talent/apiv4beta1"
+	"google.golang.org/api/iterator"
+	talentpb "google.golang.org/genproto/googleapis/cloud/talent/v4beta1"
+)
+
+// [START job_search_create_job]
+
+// createJob create a job as given.
+func createJob(w io.Writer, projectId string, jobToCreate *talentpb.Job) (*talentpb.Job, error) {
+	ctx := context.Background()
+
+	// Initialize a jobService client.
+	c, err := talent.NewJobClient(ctx)
+	if err != nil {
+		fmt.Printf("talent.NewJobClient: %v", err)
+		return nil, err
+	}
+
+  // Construct a createJob request.
+	req := &talentpb.CreateJobRequest{
```

Comment:
If you were to accept values for each important/required field on the object, then constructing the request might look like...

Note: this disregards the actual lines of code below, so this suggestion does not work as-is

```suggestion
	req := &talentpb.CreateJobRequest{
		Parent: "projects/" + projectId,
		CompanyName: companyName,
		RequisitionId: requisitionId,
		Title: title,
		Description: description,
		Addresses: []string{ address1, address2 }
	}
```

Reply:
This is an example like this in `custom_attributes_sample.go`.